tests/rptest/services/redpanda_cloud.py

```diff
@@ -613,119 +792,36 @@ def create(self, superuser: Optional[SaslCredentials] = None) -> str:
-                self.current.region)
-            # Call CloudV2 API to determine Product ID
-            self.current.product_name = self._get_product_name(
-                self.config.config_profile_name)
-            if self.current.product_name is None:
-                raise RuntimeError("ProductID failed to be determined for "
-                                   f"'{self.config.provider}', "
-                                   f"'{self.config.type}', "
-                                   f"'{self.config.install_pack_ver}', "
-                                   f"'{self.config.region}'")
-
-            # Call Api to create cluster
-            self._logger.info(f'creating cluster name {self.current.name}')
-            # Prepare cluster payload block
-            _body = self._create_cluster_payload()
-            self._logger.debug(f'body: {json.dumps(_body)}')
-            # Send API request to create cluster
-            r = self.cloudv2._http_post(
-                endpoint='/api/v1/workflows/network-cluster', json=_body)
-
-            # handle error on CloudV2 side
-            if r is None:
-                raise RuntimeError(self.cloudv2.lasterror)
-
-            try:
-                # At this point cluster has UUID instead of normal one
-
-                # For BYOC creation a non-uuid is needed
-                # It gets updated when spec makes it through
-                # the workslow, so just wait
-
-                # For FMC, we just make sure that cluster is created
-                # in API and its status is updated
-                _cluster_id = self._wait_for_cluster_id(r['id'])
-                c = self._get_cluster(_cluster_id)
-                self.current.last_status = c['state']
-            except Exception as e:
-                self.log.error()
-                raise RuntimeError(
-                    "# ERROR: Failed to get initial cluster spec")
-
-            # In case of BYOC cluster, do some additional stuff to create it
-            if self.config.type == CLOUD_TYPE_BYOC:
-                # Handle byoc creation
-                # Login without saving creds
-                self.utils.rpk_cloud_login(self.config.oauth_client_id,
-                                           self.config.oauth_client_secret)
-                # Install proper plugin
-                self.utils.rpk_cloud_byoc_install(_cluster_id)
-                # save cluster id so we can delete it in case of failure
-                self.config.id = _cluster_id
-                # Kick off cluster creation
-                # Timeout for this is half an hour as this is only agent
-                self.utils.rpk_cloud_apply(_cluster_id)
-            elif self.config.type == CLOUD_TYPE_FMC:
-                # Nothing to do here
-                pass
+            # Check if cluster is healthy
+            if not self._ensure_cluster_health(superuser):
+                self.log.warning(f"WARNING: Cluster with id '{self.config.id}' "
```

Comment:
```suggestion
                self._logger.warn(f"Cluster with id '{self.config.id}' "
```
i don't think `self.log` is available. also, `WARNING` is already prefixed in the log output when `warn()` is called.

Reply:
Got it. Will fix